### DISCUSSION TITLE
Reload of configuration

### DISCUSSION THREAD

**Discussion by mikromas** [fluent/fluentd#4298] [General]
Hi ,

Is there any alternative, other than RPC /api/config.gracefulReload and /api/config.reload, to reload the configuration of fluentd?
Thank you in advance.

**Comment by cosmo0920** [comment]
If you use non-Windows platforms, you can use SIGUSR2 or SIGHUP signals instead of the RPC endpoints.

see: https://docs.fluentd.org/deployment/signals
